docs/features/contributor-insights.md

```diff
@@ -35,4 +38,23 @@ Once your Contributor Insight Page is created, you'll be able to see an overview
 
 :::
 
-By default, your Contributor Insight Page is public. If your Contributor Insight Page is private (a PRO plan feature), only those with access to your Workspace will be able to view it.
\ No newline at end of file
+By default, your Contributor Insight Page is public. If your Contributor Insight Page is private (a PRO plan feature), only those with access to your Workspace will be able to view it.
+
+## Open Source Contributor Rating (OSCR)
+
+On your Contributor Insight Page, you'll see that each contributor is assigned an OSCR. The OSCR is a metric to evaluate the engagement and impact of contributors across the entire open source ecosystem. This score aims to give insight into a contributor's involvement and the value they bring to open source communities as a whole.
+
+The OSCR takes into account various factors related to all of the contributor's activity and the quality of their contributions in open source. OSCRs are calculated on a rolling 30-day basis. This means the score reflects recent activity and engagement, providing a current snapshot of a contributor's open source involvement rather than a lifetime aggregate.
```

Comment:
Everything looks great, my only suggestion would be to highlight this part a bit more since it's the main point that could cause confusion.

Maybe by adding an admonition or making the 30-day part bold:
```

:::info

OSCRs are calculated on a rolling 30-day basis. This means the score reflects recent activity and engagement, providing a current snapshot of a contributor's open source involvement rather than a lifetime aggregate.

:::
```